dali/operators/generic/erase/erase.cu

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <vector>
+#include "dali/core/static_switch.h"
+#include "dali/kernels/erase/erase_gpu.h"
+#include "dali/kernels/kernel_manager.h"
+#include "dali/operators/generic/erase/erase.h"
+#include "dali/operators/generic/erase/erase_utils.h"
+#include "dali/pipeline/data/views.h"
+
+#define ERASE_SUPPORTED_NDIMS (1, 2, 3, 4, 5)
+#define ERASE_SUPPORTED_TYPES (uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, \
+                               uint64_t, int64_t, float)
+namespace dali {
+
+namespace detail {
+
+template <int ndim>
+ivec<ndim> to_ivec(const TensorShape<ndim> &shape) {
+  ivec<ndim> result;
+  for (int d = 0; d < ndim; d++) {
+    result[d] = shape[d];
+  }
+  return result;
+}
```

Comment:
There is already the same function in `erase_gpu.h`.

Reply:
done